libs/blocks/marquee/marquee.js

```diff
@@ -114,4 +114,13 @@ export default function init(el) {
       media?.lastChild.remove();
     }
   }
+  if (el.classList.contains('mnemonic-list')) {
+    import('./mnemonic-list.js')
+      .then((module) => {
+        module.decorateMnemonicList(foreground);
+      })
+      .catch((err) => {
+        console.error('Failed to load module: ', err);
```

Comment:
LANA logs should be used instead of console ones

Reply:
Thanks for the heads up.